src/Microsoft.ML.ImageAnalytics/ExtensionsCatalog.cs

```diff
@@ -97,6 +97,53 @@ internal static ImageLoadingEstimator LoadImages(this TransformsCatalog catalog,
+        /// This column's data type will be <see cref="VectorDataViewType"/>.</param>
+        /// <param name="inputColumnName">Name of the column with paths to the images to load.
+        /// This estimator operates over text data.</param>
+        /// <param name="imageFolder">Folder where to look for images.</param>
+        /// <param name="useImageType">Image type flag - If true loads image as a ImageDataViewType type else loads image as VectorDataViewType. Defaults to ImageDataViewType if not specified or is true.</param>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        ///  [!code-csharp[LoadImages](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/LoadImages.cs)]
+        /// ]]></format>
+        /// </example>
+        public static ImageLoadingEstimator LoadImages(this TransformsCatalog catalog, string outputColumnName, string imageFolder, bool useImageType, string inputColumnName = null)
+           => new ImageLoadingEstimator(CatalogUtils.GetEnvironment(catalog), imageFolder, useImageType, new[] { (outputColumnName, inputColumnName ?? outputColumnName) });
+
+        /// <summary>
+        /// Loads the images from the <see cref="ImageLoadingTransformer.ImageFolder" /> into memory.
+        /// </summary>
+        /// <remarks>
+        /// The image get loaded in memory as a <see cref="VectorDataViewType" /> of bytes.
+        /// Loading is the first step of almost every pipeline that does image processing, and further analysis on images.
+        /// The images to load need to be in the formats supported by <see cref = "VectorDataViewType" /> of bytes.
+        /// For end-to-end image processing pipelines, and scenarios in your applications, see the
+        /// <a href="https://github.com/dotnet/machinelearning-samples/tree/master/samples/csharp/getting-started"> examples in the machinelearning-samples github repository.</a>
+        /// </remarks>
+        /// <param name="catalog">The transform's catalog.</param>
+        /// <param name="imageFolder">Folder where to look for images.</param>
+        /// <param name="useImageType ">Image type flag - If true loads image as a ImageDataViewType type else loads image as VectorDataViewType. Defaults to ImageDataViewType if not specified or is true.</param>
+        /// <param name="columns">Specifies the names of the input columns for the transformation, and their respective output column names.</param>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        ///  [!code-csharp[LoadImagesAsBytes](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/LoadImages.cs)]
+        /// ]]></format>
+        /// </example>
+        [BestFriend]
+        internal static ImageLoadingEstimator LoadImages(this TransformsCatalog catalog, string imageFolder, bool useImageType, params InputOutputColumnPair[] columns)
```

Comment:
Why do we have this internal overload? Why can't callers of this just use the above `public` method?

Reply:
A lot of programs were calling this function. I didn't want to change the existing function definition.

Reply:
What programs? You are adding this method in this PR. How can a lot of programs be calling this function when it didn't exist?

Reply:
I seem to have mis-read your previous comment. We don't need the internal overload. I just added it because original LoadImages() function had the internal function. I will change that.

Reply:
Removed in my recent commit.